### DISCUSSION TITLE
Discrepancies' between preview and rendered md file

### DISCUSSION THREAD

**Discussion by muddin21** [academicpages/academicpages.github.io#1969] [Q&A]
# Footnotes and alerts not working properly
Hi All,
I appreciate your assistance with following issue:
https://github.com/academicpages/academicpages.github.io/pull/1962#issue-2167450219

**Comment by rjzupkoii** [comment]
This might be more appropriate as a bug report in [Issues](https://github.com/academicpages/academicpages.github.io/issues), but there are some differences in how Markdown is rendered between the various implementations. What are you expecting to see in the deployed website?

**Comment by muddin21** [comment]
I would to see as previewed in Github which is shown below(which I would like to see after deployment):
<img width="617" alt="preview" src="https://github.com/academicpages/academicpages.github.io/assets/106573604/8ee50c7d-9e91-48c2-90f3-3b7ee859112c">
however, it's shown below after the deployment
<img width="643" alt="render" src="https://github.com/academicpages/academicpages.github.io/assets/106573604/f3a1bc13-a133-4780-86f5-ebee8350b5ac">

**Reply by rjzupkoii** [reply]
Is the issue that the footnotes are not being rendered at all when you deploy the website, or that they aren't formatted the same as they are displayed on GitHub? Since footnotes are part of the [extended Markdown syntax](https://www.markdownguide.org/extended-syntax/#footnotes) it may be a feature request to add them to the template if they aren't already. However, if they are rendering, but don't match the style of GitHub then that's more of a styling issue since the style applied when Markdown is rendered isn't guaranteed to be the same between implementations.

**Comment by muddin21** [comment]
Footnotes are being rendered after deployment, but the format is not the same as they are displayed on GitHub. I attached the rendered version of a footnote in the below of the above comment.

**Reply by rjzupkoii** [reply]
Since the `[^1]` tag is present in the screenshot, it looks like the Markdown wasn't rendered. I'm going to go ahead and covert this to a bug report under [Issues](https://github.com/academicpages/academicpages.github.io/issues) but depending upon the cause it might take a little while to address the problem.
